binr/ragg2/ragg2.c

```diff
@@ -108,6 +123,420 @@ static int openfile (const char *f, int x) {
 }
 #define ISEXEC (fmt!='r')
 
+static char* getCompiler(void) {
+	char *output = NULL;
+	char *CC = NULL;
+
+	output = r_sys_getenv ("CC");
+	// if CC is not set, or CC is not gcc, llvm-gcc, or clang
+	if (!output || (strcmp (output, "llvm-gcc") 
```

Comment:
remove the comment and move the logic of this conditional to a function that gets output as parameter with a meaningful name